react/Section.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from 'react'
+import { useCssHandles, applyModifiers } from 'vtex.css-handles'
+import { useResponsiveValue } from 'vtex.responsive-values'
+
+interface Props {
+  borderless?: MaybeResponsiveInput<boolean>
+  marginBottom?: MaybeResponsiveInput<number>
+  paddingBottom?: MaybeResponsiveInput<number>
+  width?: MaybeResponsiveInput<string>
+  name: string
+}
+
+const CSS_HANDLES = ['section']
+
+const Section: FC<Props> = ({
+  children,
+  borderless: borderlessProp = false,
+  marginBottom: marginBottomProp,
+  paddingBottom: paddingBottomProp,
+  width: widthProp,
+  name,
+}) => {
+  const handles = useCssHandles(CSS_HANDLES)
+
+  const borderless = useResponsiveValue(borderlessProp)
+  const width = useResponsiveValue(widthProp)
+  const marginBottom = useResponsiveValue(marginBottomProp)
+  const paddingBottom = useResponsiveValue(paddingBottomProp)
```

Comment:
You could use `useResponsiveValueS` to get all of them out of props. There is an example in the docs https://github.com/vtex-apps/responsive-values

Reply:
DAMN YOU MISSING TYPES